src/fmu/dataio/_models/fmu_results/data.py

```diff
@@ -47,7 +44,7 @@ class Time(BaseModel):
 
     .. note:: ``data.time`` items can currently hold a maximum of two values."""
 
-    t0: Optional[Timestamp] = Field(default=None)
+    t0: Timestamp
```

Comment:
Note, `data.time` is still **optional**. This change here just requires `t0` to be set if `data.time` is set.